test/asm/test.sh

```diff
@@ -59,7 +59,7 @@ fi
 
 # Check whether to use '.simple.err' files if they exist
 # (rgbasm with pre-3.0 Bison just reports "syntax error")
-$RGBASM -Weverything -o "$o" syntax-error.asm >"$output" 2>"$errput"
+$RGBASM -Weverything -X 0 -o "$o" syntax-error.asm >"$output" 2>"$errput"
```

Comment:
I like this idea! It could even justify lowering the default error limit for manual invocations -- but maybe not, since everyone's currently used to the unlimited behavior; let's see how 100 goes.